go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/onsi/gomega v1.29.0
```

Comment:
unrelated: headsup that gomega has associated CVEs on their library (been bumping it in all toolkit packages in aks repo).
I would suggest to bumpt to latest

Reply:
I don't think we use gomega in the feature code

Reply:
Still shows up in the repo scans, because transitive dependencies show up in the go.sum
we have to fix them all in AKS internal repos. But up to you if you don't want to follow that guidance.